### ISSUE TITLE
Parse and Render helper methods?

### ISSUE THREAD

**Issue by oconnor0** [xledger/Xledger.Sql#7]
Might be nice to include Parse (or TryParse) and Render helper methods for the immutable DOM. The implementation would likely involved conversion to/from the mutable DOM so would probably be "slow".

**Comment by isaksky** [comment]
Hmm maybe. But sometimes the absence of such functions can be instructive. In the same way there is no `List.nth` in F# lists (linked).

I guess that problem can also be solved with the function name though, parseAndConvert or something.

**Comment by oconnor0** [comment]
Agreed. I ran into a place while debugging when Render would have been helpful, but there's a cost there so figured it'd be worth thinking whether there's a right way to do it.
